website/docs/api/span-resolver.md

```diff
@@ -0,0 +1,353 @@
+---
+title: SpanResolver
+tag: class,experimental
+source: spacy-experimental/coref/span_resolver_component.py
+teaser: 'Pipeline component for resolving tokens into spans'
+api_base_class: /api/pipe
+api_string_name: span_resolver
+api_trainable: true
+---
+
+> #### Installation
+>
+> ```bash
+> $ pip install -U spacy-experimental
+> ```
+
+<Infobox title="Important note" variant="warning">
+
+This component not yet integrated into spaCy core, and is available via the
+extension package
+[`spacy-experimental`](https://github.com/explosion/spacy-transformers) starting
+in version 0.6.0. It exposes the component via
+[entry points](/usage/saving-loading/#entry-points), so if you have the package
+installed, using `factory = "span_resolver"` in your
+[training config](/usage/training#config) or `nlp.add_pipe("span_resolver")`
+will work out-of-the-box.
+
+</Infobox>
+
+A `SpanResolver` component takes in tokens (represented as `Span`s of length 1)
+and resolves them into `Span`s of arbitrary length. The initial use case is as a
+post-processing step on word-level [coreference resolution](/api/coref). The
+input and output keys used to store `Span`s are configurable.
```

Comment:
To avoid the slightly awkward "`Span`'s" I'd prefer something like "`Span` objects"

Reply:
Sounds good, changed.